### PR TITLE
gnome-extra/chrome-gnome-shell: version bump, remove old

### PR DESCRIPTION
**Pull request by nE0sIghT**
https://mail.gnome.org/archives/ftp-release-list/2016-September/msg00019.html
https://wiki.gnome.org/Projects/GnomeShellIntegrationForChrome/ReleaseNotes/7

### CODE REVIEW

**Comment by nE0sIghT** [comment]
Please do not merge for now.
Package need some important change

**Comment by nE0sIghT** [comment]
Fixed version pushed

File: gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild [hunk 1]
```diff
@@ -19,15 +19,26 @@ IUSE=""
...
+DEPEND="${PYTHON_DEPS}
+	sys-apps/coreutils
+"
 RDEPEND="${PYTHON_DEPS}
 	dev-python/pygobject:3[${PYTHON_USEDEP}]
 	gnome-base/gnome-shell
 "
 
 src_configure() {
 	python_setup
```
Comment (SoapGentoo) [inline gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild]: dont't need this, `python-single-r1` exports its `pkg_setup` which does the setup.

File: gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild [hunk 2]
```diff
@@ -19,15 +19,26 @@ IUSE=""
...
-	python_fix_shebang ./connector
 
 	local mycmakeargs=( -DBUILD_EXTENSION=OFF )
 	cmake-utils_src_configure
 }
+
+src_install() {
+	echo -n "CONFIG_PROTECT_MASK=\"" > 50"${PN}"
+	echo -n "/etc/chromium/policies/managed/${PN}.json " >> 50"${PN}"
+	echo "/etc/opt/chrome/policies/managed/${PN}.json\"" >> 50"${PN}"
```
Comment (SoapGentoo) [inline gnome-extra/chrome-gnome-shell/chrome-gnome-shell-7.ebuild]: as soon as you do filesystem I/O with external commands, you need `|| die` on them

**Comment by SoapGentoo** [comment]
LGTM @gentoo/proxy-maint

**Comment by nE0sIghT** [comment]
Ping

**Pull request closed without merging**